lib/howl/buffer.moon

```diff
@@ -295,9 +305,15 @@ class Buffer extends PropertyObject
     @viewers -= 1
 
   _associate_with_file: (file) =>
+    scope = @_config_scope
     @_file = file
+    config.copy scope, @_config_scope
```

Comment:
I still think this seems like it's clobbering things, but perhaps I'm squarely in theoretical county here. However, say I have a buffer open with a config and no file, which I then save to `foo.moon`. The current code would just remove any existing configuration for the `file/foo.moon` and replace that with the current buffer config. That doesn't seem right? In the future I might have a particular config setup for this particular file, which would then be destroyed.